### DISCUSSION TITLE
Time into mailer log would not be correct

### DISCUSSION THREAD

**Discussion by ruditimmermans** [vit1251/golden#167] [Q&A] [answered]
When i check my mailer log the time show there would not be correct. Here would be a debug log

**Comment by vit1251** [comment]
Look like your mailer log stay at UTC and mailer also use UTC time

```
2022/06/29 18:30:14.248109 writeCommandPacket: commandID = "M_NUL" body = TIME 29 Jun 22 18:30 UTC
2022/06/29 18:30:14.248134 writeCommandPacket: commandID = "M_NUL" body = OS Android
```

but remote side use is +0200 zone time

```
2022/06/29 18:30:14.249478 MailerStream: RX frame: commandID = "M_NUL" body = TIME Wed, 29 Jun 2022 20:30:19 +0200
2022/06/29 18:30:14.249511 Mailer: Remote side M_NUL packet: name = TIME value = Wed, 29 Jun 2022 20:30:19 +0200
```

Look like your android version unable to determine user valid timezone.

**Comment by ruditimmermans** [comment]
Oh i see not that big deal it works....

**Comment by vit1251** [comment]
Could you start GOlden point with "--debug" options and check head lines.
You may see  debug message about your system timezone like "Time zone: MSK (10800)"

**Comment by ruditimmermans** [comment, marked as answer]
It's been fixed it was a bug into Termux.... BTW i have also now a Chromebook with Golden running on that into the Linux container ;)
